### DISCUSSION TITLE
Function working on deployed version

### DISCUSSION THREAD

**Discussion by guess01325** [vercel/community#719] [Help] [answered]
Hi, my name is Otis, I am new here, and new at coding. The reason I am writing this is because I deployed a ruby rails and react app. I can sign on in the deployed version, but not if I run it locally. This would cause problems when I am trying to test things out on local branch. Does anyone know what this might be?

**Comment by swarnava** [comment, marked as answer]
Sounds like an issue with CORS or something. Are you getting any error in Console?

**Comment by guess01325** [comment]
getting a 404 post request and axios error

**Reply by swarnava** [reply]
Likely not related to Vercel then.

**Comment by amyegan** [comment]
Hi @guess01325. I think Swarnava is right. This does sound like it's a problem with CORS or something similar. Since it is working on Vercel but not local, I don't think we're going to be able to help you as much as we would like. You might get suggestions more specific to your app by asking in communities devoted to the specific tech stack you're using. For example:
- [Axios discussions](https://github.com/axios/axios/discussions) might be able to help
- [Ruby community](https://www.ruby-lang.org/en/community/) has user groups and a [Ruby Discord server](https://discord.gg/EnSevaRfct)
- [React community support](https://reactjs.org/community/support.html) suggests a few forums where you might get some good advice
